src/Mhwd.cpp

```diff
@@ -552,6 +552,7 @@ MHWD::STATUS Mhwd::installConfig(std::shared_ptr<Config> config)
 MHWD::STATUS Mhwd::uninstallConfig(Config *config)
 {
     std::shared_ptr<Config> installedConfig{getInstalledConfig(config->name_, config->type_)};
+    std::vector<std::shared_ptr<Config>> localRequirements{data_.getAllLocalRequirements(installedConfig)};
```

Comment:
Local requirements are checked before this function is called. We don't need that check here.